doc/Makefile.am

```diff
@@ -196,12 +196,12 @@ geany-sciwrappers-gtkdoc-tmp.h: geany-sciwrappers-gtkdoc.h
 		-e 's,sci_,scintilla_object__GI__MARK_,g' \
 		$< > $@
 
-GeanyScintilla-1.0.gir: GeanyScintilla-1.0-pre.gir
+GeanyScintilla-1.0.gir: GeanyScintilla-1.0-tmp.gir
 	$(AM_V_GEN) cat $< \
```

Comment:
@kugel- would need to answer, but I'd guess it's because `g-ir-scanner` is called by the generic introspection makefile rather than implemented manually, so we can't easily sed its output.
If we can, that'd be nicer indeed.